driver/utils/utils.h

```diff
@@ -126,6 +110,37 @@ struct UTF8CaseInsensitiveCompare {
     }
 };
 
+template<typename T>
+class ObjectPool {
```

Comment:
Could you please add a comment describing that this pool is for, and to document that if behaves in LRU-fashion by dropping old object after reaching certain size.

Reply:
ok